src/Compatibility/ApiCompat/Microsoft.DotNet.ApiCompatibility/Resources.resx

```diff
@@ -186,8 +186,8 @@
   <data name="CannotAddVirtualToMember" xml:space="preserve">
     <value>Cannot add virtual keyword to member '{0}'.</value>
   </data>
-  <data name="CannotRemoveVirtualFromMember" xml:space="preserve">
-    <value>Cannot remove virtual keyword from member '{0}'.</value>
+  <data name="CannotRemoveVirtualOrAbstractFromMember" xml:space="preserve">
+    <value>Cannot remove '{0}' keyword from member '{1}'.</value>
```

Comment:
Might need to update docs: https://learn.microsoft.com/en-us/dotnet/fundamentals/apicompat/diagnostic-ids

Reply:
https://github.com/dotnet/docs/pull/44889

Reply:
Yeah, I think we can just update those to mention either case.